tests/testthat/test-get_map_points.R

```diff
@@ -24,4 +23,5 @@ test_that("get_map_points works", {
                                            point_id = "test"),
                           warn_args = list(point_id = -1),
                           col_no = 9)
+  expect_true(is.data.frame(get_map_points(point_id = 1, sf = FALSE)))
```

Comment:
If you expect a different number of columns for `sf = TRUE` vs `sf = FALSE`, I would run a whole second suite of tests like in this test file: https://github.com/palaeoverse/rmacrostrat/blob/main/tests/testthat/test-def_drilling_sites.R

Reply:
I am not sure this is really needed? We are pulling the exact same data and format, just converting it to a `data.frame`? Isn't this unnecessarily repeating tests? What am I missing?

Reply:
Oh, ok, I guess this is fine then! In that case, I would suggest also checking that it _isn't_ an `sf` object (since `sf` objects are also `data.frame`s).

Reply:
😉 Fair